File: 백준/Silver/2563. 색종이/색종이.py
import sys

N = int(sys.stdin.readline().rstrip())

paper = [[0 for _ in range(100)] for _ in range(100)]

for _ in range(N):
    x, y = map(int, sys.stdin.readline().strip().split())
    for i in range(x, x + 10):
        for j in range(y, y + 10):
            paper[i][j] = 1

area = sum(sum(row) for row in paper)

sys.stdout.write(str(area) + '\n')

File: 백준/Silver/2563. 색종이/색종이.java
import java.io.*;
import java.util.*;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

		int[][] paper = new int[100][100];

		int N = Integer.parseInt(br.readLine());
		for (int k = 0; k < N; k++) {
			StringTokenizer st = new StringTokenizer(br.readLine());
			int x = Integer.parseInt(st.nextToken());
			int y = Integer.parseInt(st.nextToken());
			for (int i = x; i < x + 10; i++) {
				for (int j = y; j < y + 10; j++) {
					paper[i][j] = 1;
				}
			}
		}

		int area = 0;

		for (int i = 0; i < 100; i++) {
			for (int j = 0; j < 100; j++) {
				area += paper[i][j];
			}
		}

		bw.write(String.valueOf(area) + "\n");

		br.close();
		bw.flush();
		bw.close();
	}
}